mlflow/utils/search_utils.py

```diff
@@ -211,7 +211,12 @@ def _valid_entity_type(cls, entity_type):
     @classmethod
     def _get_identifier(cls, identifier, valid_attributes):
         try:
-            entity_type, key = identifier.split(".", 1)
+            tokens = identifier.split(".", 1)
+            if len(tokens) == 1:
+                key = tokens[0]
+                entity_type = cls._ATTRIBUTE_IDENTIFIER
+            else:
+                entity_type, key = tokens
```

Comment:
@dbczumar This is a change to support a key-only filter string such as `run_id IN (...)`.